macros/utils/data_types/get_data_type_synonyms.sql

```diff
@@ -0,0 +1,107 @@
+{% macro get_data_type_synonyms(exact_data_type) %}
```

Comment:
Cosmetic note - "get_data_type_synonyms" sounds a bit like you return a list of synonyms, which is not what happens.
How about calling it something like "normalize_data_type"?

Reply:
Good comment, changing

Reply:
renamed to get_normalized_data_type (left the "get_" )